examples/maskrcnn-example/utils/encord_dataset.py

```diff
@@ -21,6 +21,16 @@ def __getitem__(self, idx):
 
         boxes, labels, area, iscrowd = [], [], [], []
         for target_item in target:
+            if (not target_item["bbox"][2] > 0) or (not target_item["bbox"][3] > 0):
+                print(
```

Comment:
💭 probably best if added as a log to be able to silence it

Reply:
I think we should also warn users that there are annotations like that so that they can fix/remove those. What do you think is the best way to give a message to the user?

Reply:
@eloy-encord

Reply:
I would go with logs as we have done before in EA, or an optional `verbose` argument that if set to `True` it will show the warnings.